File: 4 Jan 2026 1390. Four Divisors/code.py
class Solution:
    def sumFourDivisors(self, nums: List[int]) -> int:
        res=0
        for i in nums:
            cnt,f=0,0
            for j in range(2,int(sqrt(i))+1):
                if(i%j==0):cnt,f=cnt+1,j
                if(cnt>1):break
            if(cnt==1):
                if(f==(i/f)):continue
                res=res+1+i+f+(i/f)
        return int(res)

File: 15 Jan 2026 2943. Maximize Area of Square Hole in Grid/code.cpp
class Solution {
public:
    int rec(vector<int>& nums) {
        int s=1,cnt=1;
        for (int i=0;i<nums.size() - 1;i++) {
            if (nums[i + 1]-nums[i]== 1) {
                cnt++;
            } else {
                s=max(s,cnt + 1);
                cnt=1;
            }
        }
        s = max(s,cnt+1);
        return s;
    }
    int maximizeSquareHoleArea(int n, int m, vector<int>& hBars, vector<int>& vBars) {
        sort(hBars.begin(), hBars.end());
        sort(vBars.begin(), vBars.end());
        int hc=rec(hBars),vc=rec(vBars);
        int a=min(hc,vc);
        return a*a;
    }
};
